api/desecapi/tests/test_rrsets.py

```diff
@@ -344,6 +344,16 @@ def test_update_essential_properties(self):
         self.assertStatus(response, status.HTTP_400_BAD_REQUEST)
         self.assertEquals(response.data['type'][0].code, 'read-only-on-update')
 
+        # Changing "created" is no-op
+        response = self.client.get(url)
+        data = response.data
+        created = data['created']
+        data['created'] = '2019-07-19T17:22:49.575717Z'
+        response = self.client.patch(url, data)
+        self.assertStatus(response, status.HTTP_200_OK)
+        response = self.client.patch(url, data)
+        self.assertStatus(response, status.HTTP_200_OK)
```

Comment:
Why patching twice?

Reply:
I guess one should have been PUT, shall change